broker/client/client.go

```diff
@@ -789,7 +800,7 @@ func (c *Iso18626Client) sendAndUpdateSupplier(ctx common.ExtendedContext, trCtx
 	resData := events.EventResult{}
 	resData.OutgoingMessage = message
 	if !isDoNotSend(trCtx.event) {
-		response, err := c.SendHttpPost(trCtx.selectedPeer, message)
+		response, err := c.SendIllMessage(ctx, trCtx.selectedPeer, message)
```

Comment:
@JanisSaldabols the isDoNotSend is used when we have the so-called localSupply (we should rename to selfSupply or something) and we need to consider whether this should be handled differently for local peers.